data/steam

```diff
@@ -9,17 +9,34 @@ steamstatic.com
 steamusercontent.com
 underlords.com
 valvesoftware.com
-wmsjsteam.com @cn
 
+# 完美
 csgo.wmsj.cn @cn
-dl.steam.ksyna.com @cn
 dota2.wmsj.cn @cn
+wmsjsteam.com @cn
+
+# 金山云
+dl.steam.clngaa.com @cn
+dl.steam.ksyna.com @cn
+
+# 白山云
 st.dl.bscstorage.net @cn
 st.dl.eccdnx.com @cn
 st.dl.pinyuncloud.com @cn
+
+# 华为云
 steampipe.steamcontent.tnkjmec.com @cn
+
+# 网宿
 steampowered.com.8686c.com @cn
 steamstatic.com.8686c.com @cn
+cdn-ws.content.steamchina.com
+
+# 腾讯云（仅在蒸汽中国，但可用于国际版）
+cdn-qc.content.steamchina.com
+
+# 阿里云（仅在蒸汽中国，但可用于国际版）
+cdn-ali.content.steamchina.com
```

Comment:
Should `steamchina.com` be marked with `@cn`?

Reply:
good call